cmd/kubetype-gen/testdata/test_output/defaults/group/version/doc.go

```diff
@@ -17,4 +17,5 @@
 // Package has auto-generated kube type wrappers for raw types.
 // +k8s:openapi-gen=true
 // +k8s:deepcopy-gen=package
+// +groupName=group
```

Comment:
Do you know why groupName is generated here?

Reply:
No, I don't see where that came from...